wp-admin/css/colors/_admin.scss

```diff
@@ -516,10 +499,20 @@ div#wp-responsive-toggle a:before {
 	background: $menu-highlight-background;
 }
 
-.star-rating .star {
-	color: $highlight-color;
-}
-
 .wp-responsive-open #wpadminbar #wp-admin-bar-menu-toggle a {
 	background: $menu-submenu-background;
 }
+
+.wp-responsive-open #wpadminbar #wp-admin-bar-menu-toggle .ab-icon:before {
```

Comment:
Avoid using id selectors<br>Begin pseudo elements with double colons: `::`<br>Selector should have depth of applicability no greater than 2, but was 4